src/client/Microsoft.Identity.Client/ITokenCacheInternal.cs

```diff
@@ -66,7 +64,7 @@ internal interface ITokenCacheInternal : ITokenCache, ITokenCacheSerializer
         /// Shows if MSAL's in-memory token cache has any kind of RT or non-expired AT. Does not trigger a cache notification.
         /// Ignores ADAL's cache.
         /// </summary>
-        bool HasTokensNoLocks();
+        bool HasTokensNoLocks(string partitionKey);
```

Comment:
Not sure if this should be partitioned, i.e. 

- HasTokensNoLocks(null) -> are there any tokens at all?
- HasTokensNoLocks(partitionKey) -> are there any tokens in the partition?

I don't think we would use the second variant in any code path. We only care about total tokens.

Reply:
i think this one should move to `IAccessor` where we can optimize it a bit.

Reply:
Updated. Removed partition key parameter. Moved it to the accessor. I did end up changing the [iOS]() and [Android]()